File: app/src/main/java/xh/org/video/android_ppjoke/utils/StringConvert.java
package xh.org.video.android_ppjoke.utils;

public class StringConvert {
    public static String convertFeedUgc(int value) {
        if (value < 10000) {
            return String.valueOf(value);
        }
        return value / 10000 + "万";
    }
}

File: app/src/main/java/xh/org/video/android_ppjoke/utils/StringConvert.kt
package xh.org.video.android_ppjoke.utils

class StringConvert {
    companion object {
        fun convertFeedUgc(count: Int): String? {
            return if (count < 10000) {
                count.toString()
            } else (count / 10000).toString() + "万"
        }
    }

}